jobs/e2e-test.groovy

```diff
@@ -302,6 +305,19 @@ def runTestServer() {
       // before it does so, so it doesn't read "old" data.
       sh("rm -rf genfiles/test-reports");
 
+      tests = sh(script: "testing/runtests_server.py -n ${exec.shellEscapeList(runSmokeTestsArgs)} . ", returnStdout: true);
+      // The runtests_server.py script with -n outputs all tests to run of 
+      // various types. We only need to worry about smoke tests, which are 
+      // also the last section, so grab everything after the header. If it's 
+      // empty, there are no tests to run.
+      e2eTests = tests.substring(tests.lastIndexOf("SMOKE TESTS:") + 13).trim()
+      echo("e2eTests: ${e2eTests}")
```

Comment:
Did you mean to keep this in?

Reply:
I did, as it could definitely make the debugging easier, but the logs are darn noisy already so if we want to eliminate it we certainly could.